From 51f4c18f0857473f864771c55c8caea200c3b255 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 1 Jun 2022 16:14:14 +0800
Subject: [PATCH 16/31] Revert "drm/amdkfd: add missing void argument to
 function kgd2kfd_init"

This reverts commit 1549bc8cc165800842f281a48a28ae7087e1053e.
---
 drivers/gpu/drm/amd/amdkfd/kfd_module.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_module.c b/drivers/gpu/drm/amd/amdkfd/kfd_module.c
index f4b7f7e6c40e..986ff52d5750 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_module.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_module.c
@@ -82,7 +82,7 @@ static void kfd_exit(void)
 	kfd_chardev_exit();
 }
 
-int kgd2kfd_init(void)
+int kgd2kfd_init()
 {
 	return kfd_init();
 }
-- 
2.27.0

